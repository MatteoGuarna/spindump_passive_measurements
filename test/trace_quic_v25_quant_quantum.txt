Test quantum-readiness with QUIC/Quant (multiple initial packets)
