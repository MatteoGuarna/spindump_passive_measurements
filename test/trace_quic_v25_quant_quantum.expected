QUIC 100.72.46.172 <-> 13.48.145.10 92220f8fd32d23bae6043b6f-null (52633:4433) at 1580745692805247 new starting packets 0 0 bytes 0 0 note "No rsp,v.0xbabababa,no spin"
QUIC 100.72.46.172 <-> 13.48.145.10 92220f8fd32d23bae6043b6f-null (52633:4433) at 1580745692805248 packet starting initiator length 1228 packets 2 0 bytes 2508 0 bandwidth 2508 0 note "No rsp,v.0xbabababa,no spin"
QUIC 100.72.46.172 <-> 13.48.145.10 92220f8fd32d23bae6043b6f-3b6f31ea099222 (52633:4433) at 1580745692839586 change starting packets 2 0 bytes 2508 0 bandwidth 2508 0 note "No rsp,v.0x00000000,no spin"
QUIC 100.72.46.172 <-> 13.48.145.10 92220f8fd32d23bae6043b6f-3b6f31ea099222 (52633:4433) at 1580745692839586 measurement starting right 34338 packets 2 0 bytes 2508 0 bandwidth 2508 0 note "No rsp,v.0x00000000,no spin"
QUIC 100.72.46.172 <-> 13.48.145.10 null-3b6f31ea099222 (52633:4433) at 1580745692839589 measurement starting right 34341 packets 2 1 bytes 2508 56 bandwidth 2508 56 note "V.0x00000000,no spin"
QUIC 100.72.46.172 <-> 13.48.145.10 null-3b6f31ea099222 (52633:4433) at 1580745692841177 measurement starting left 1588 packets 2 2 bytes 2508 112 bandwidth 2508 112 note "V.qn25,no spin"
QUIC 100.72.46.172 <-> 13.48.145.10 null-3b6f31ea099222 (52633:4433) at 1580745692841178 measurement starting left 1589 packets 3 2 bytes 3788 112 bandwidth 3788 112 note "V.qn25,no spin"
QUIC 100.72.46.172 <-> 13.48.145.10 null-044b4d83 (52633:4433) at 1580745692879037 change starting packets 4 2 bytes 5016 112 bandwidth 5016 112 note "V.qn25,no spin"
QUIC 100.72.46.172 <-> 13.48.145.10 null-044b4d83 (52633:4433) at 1580745692879037 measurement up right 37859 packets 4 2 bytes 5016 112 bandwidth 5016 112 note "V.qn25,no spin"
QUIC 100.72.46.172 <-> 13.48.145.10 9602f392-044b4d83 (52633:4433) at 1580745692879338 packet up responder length 1280 packets 4 4 bytes 5016 2589 bandwidth 5016 2589 note "V.qn25,no spin"
QUIC 100.72.46.172 <-> 13.48.145.10 9602f392-044b4d83 (52633:4433) at 1580745692880138 packet up responder length 1280 packets 4 5 bytes 5016 3869 bandwidth 5016 3869 note "V.qn25,no spin"
QUIC 100.72.46.172 <-> 13.48.145.10 9602f392-044b4d83 (52633:4433) at 1580745692880644 packet up initiator length 68 packets 5 5 bytes 5084 3869 bandwidth 5084 3869 note "V.qn25,no spin"
QUIC 100.72.46.172 <-> 13.48.145.10 9602f392-044b4d83 (52633:4433) at 1580745692880841 packet up initiator length 67 packets 6 5 bytes 5151 3869 bandwidth 5151 3869 note "V.qn25,no spin"
QUIC 100.72.46.172 <-> 13.48.145.10 9602f392-044b4d83 (52633:4433) at 1580745692882284 packet up initiator length 1500 packets 7 5 bytes 6651 3869 bandwidth 6651 3869 note "V.qn25,no spin"
QUIC 100.72.46.172 <-> 13.48.145.10 1d755ca6-044b4d83 (52633:4433) at 1580745692917099 packet up responder length 70 packets 7 6 bytes 6651 3939 bandwidth 6651 3939 note "V.qn25,no spin"
QUIC 100.72.46.172 <-> 13.48.145.10 1d755ca6-044b4d83 (52633:4433) at 1580745692917494 packet up responder length 1500 packets 7 7 bytes 6651 5439 bandwidth 6651 5439 note "V.qn25,no spin"
QUIC 100.72.46.172 <-> 13.48.145.10 047db096-85c8648b57eccb0009f0 (61916:4433) at 1580745694310772 new starting packets 0 0 bytes 0 0 note "No rsp,v.qn25,no spin"
QUIC 100.72.46.172 <-> 13.48.145.10 047db096-85c8648b57eccb0009f0 (61916:4433) at 1580745694310773 packet starting initiator length 680 packets 2 0 bytes 1960 0 bandwidth 1960 0 note "No rsp,v.qn25,no spin"
QUIC 100.72.46.172 <-> 13.48.145.10 047db096-041d5252 (61916:4433) at 1580745694348607 change starting packets 2 0 bytes 1960 0 bandwidth 1960 0 note "No rsp,v.qn25,no spin"
QUIC 100.72.46.172 <-> 13.48.145.10 047db096-041d5252 (61916:4433) at 1580745694348607 measurement up right 37834 packets 2 0 bytes 1960 0 bandwidth 1960 0 note "No rsp,v.qn25,no spin"
QUIC 100.72.46.172 <-> 13.48.145.10 7db09650-041d5252 (61916:4433) at 1580745694351019 packet up initiator length 144 packets 3 1 bytes 2104 1500 bandwidth 2104 1500 note "V.qn25,no spin"
QUIC 100.72.46.172 <-> 13.48.145.10 7db09650-041d5252 (61916:4433) at 1580745694351480 packet up initiator length 1500 packets 4 1 bytes 3604 1500 bandwidth 3604 1500 note "V.qn25,no spin"
QUIC 100.72.46.172 <-> 13.48.145.10 7db09650-041d5252 (61916:4433) at 1580745694385648 packet up responder length 70 packets 4 2 bytes 3604 1570 bandwidth 3604 1570 note "V.qn25,no spin"
QUIC 100.72.46.172 <-> 13.48.145.10 5aeabf9e-041d5252 (61916:4433) at 1580745694389560 packet up responder length 70 packets 6 9 bytes 3716 10231 bandwidth 3716 10231 note "V.qn25,no spin"
QUIC 100.72.46.172 <-> 13.48.145.10 5aeabf9e-041d5252 (61916:4433) at 1580745694423129 measurement up left 431 packets 7 11 bytes 3769 10451 bandwidth 3769 10451 note "V.qn25,spinning"
